src/app.js

```diff
@@ -1,11 +1,16 @@
 const express = require('express');
-
+const artistRouter = require('./routes/artist');
+const albumRouter = require('./routes/album');
 const app = express();
 
 app.use(express.json());
 
-app.get("/", (_req, res) => {
-  res.send("Hello World!");
+app.get('/', (_req, res) => {
+  res.send('Hello World!');
 });
 
-module.exports = app;
\ No newline at end of file
+app.use('/', artistRouter);
```

Comment:
the path here should be `"/artists"` as every route that comes of here starts with it, so it should be in here
```suggestion
app.use('/artists', artistRouter);
```